querydsl-core/src/main/java/com/querydsl/core/alias/ManagedObject.java

```diff
@@ -22,10 +22,10 @@
 public interface ManagedObject {
 
     /**
-     * Returns the underlying {@link EntityPath} instance of this proxy
+     * Returns the underlying {@link Path} instance of this proxy
      *
-     * @return underlying {@link EntityPath} instance
+     * @return underlying {@link Path} instance
      */
-    EntityPath<?> __mappedPath();
+    Path<?> __mappedPath();
```

Comment:
This was faulty. The `Path` may also be a `SetPath`, `MapPath`, `ListPath` or `CollectionPath`.